utils/fuzzer-options/src/lib.rs

```diff
@@ -226,7 +227,8 @@ pub enum Commands {
 }
 
 impl FuzzerOptions {
-    /// given an `App`, add it to FuzzerOptions as a subcommand and return the resulting `App`
+    #[must_use]
```

Comment:
I think we usually comment above the `#` stuff (unless when we don't)

Reply:
fixed